.docs/index.md

```diff
@@ -119,9 +92,12 @@ libstorage:
   service: virtualbox
 ```
 
-Refer to the
-[VirtualBox documentation](http://libstorage.readthedocs.io/en/stable/user-guide/storage-providers/#virtualbox)
-for additional VirtualBox configuration options.
+Refer to the [VirtualBox documentation](http://libstorage.readthedocs.io/en/stable/user-guide/storage-providers/#virtualbox) for additional VirtualBox configuration options.
```

Comment:
Hi @kacole2,

I've previously noted for sake of readability and long-term support it's a documented request to limit Markdown to a width of 80 columns. Please reformat the above text. Thanks!